### PR TITLE
Just some improvements on the scrollbar

### PR DESCRIPTION
**Pull request by devsnaked**
It's just an improvement on the scrollbars.
I added a CSS class called .no-scrollbar to hide the scrollbars by default and show it on hover, making it more compatible with the UI.

**Before**
scrollbar-before

**After**
scrollbar-after

My elixir extension added the parentheses in the functions when I saved, I can remove it if necessary.

### CODE REVIEW

File: elixir/apps/web/assets/css/app.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
 @import "tailwindcss/base";
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
+@import "./scrollbar.css"
```
Comment (AndrewDryga) [inline elixir/apps/web/assets/css/app.css]: A newline at the end of file is needed by our conventions.

File: elixir/apps/web/lib/web/components/core_components.ex [hunk 2]
```diff
@@ -52,9 +52,9 @@ defmodule Web.CoreComponents do
     The lazy brown fox jumped over the quick dog.
   </.code_block>
   """
-  attr :id, :string, required: true
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/components/core_components.ex]: Please rollback those changes, your editor did not pick up project formatting rules.

**Comment by AndrewDryga** [comment]
Hey @devsnaked, thank you for this PR! I left two minor comments, rest LGTM!

File: elixir/apps/web/assets/css/app.css (hunk 1 above)
Comment (jamilbk) [reply elixir/apps/web/assets/css/app.css]: Might need a semicolon at the end here too.

File: elixir/apps/web/lib/web/components/core_components.ex (hunk 2 above)
Comment (devsnaked) [reply elixir/apps/web/lib/web/components/core_components.ex]: I made the changes, now I believe that's OK.

**Comment by AndrewDryga** [comment]
Thank you @devsnaked ❤️

**Comment by AndrewDryga** [comment]
@devsnaked can you please fix the newline so that the static analysis CI step will pass? I guess it finds something like `/r` instead of `/n` or something like that, most a likely problem with the editor config. After that, I'll merge it right away :).

File: elixir/apps/web/assets/css/app.css [hunk 3]
```diff
@@ -1,3 +1,5 @@
 @import "tailwindcss/base";
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
+@import "./scrollbar";
+
```
Comment (jamilbk) [inline elixir/apps/web/assets/css/app.css:5]:
```suggestion
```

Comment (jamilbk) [reply elixir/apps/web/assets/css/app.css:5]: Just need to remove this newline and the CI should pass. Otherwise LGTM.

**Review by jamilbk — CHANGES_REQUESTED** [review]

File: elixir/apps/web/assets/css/app.css (hunk 1 above)
Comment (jamilbk) [reply elixir/apps/web/assets/css/app.css]: Sorry for the confusion -- there should be _no_ newline here @devsnaked

**Comment by jamilbk** [comment]
Superseded by #2160

**Pull request closed without merging**